contracts/src/integrations/cert/router/CertVerifierRouterFactory.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.12;
+
+import {EigenDACertVerifierRouter} from "src/integrations/cert/router/EigenDACertVerifierRouter.sol";
+
+/// @notice For use by rollups to deploy a CertVerifierRouter atomically without a proxy.
+contract CertVerifierRouterFactory {
```

Comment:
```suggestion
contract ImmutableCertVerifierRouterFactory {
```

Reply:
ah good point. I guess most use case of "immutable" ive heard just means "not behind a proxy". For example
<img width="684" height="222" alt="image" src="https://github.com/user-attachments/assets/fd6a19de-48d3-4056-8b20-9192f6966eed" />

But I get your argument as well.